Assets/Scripts/Game/EnemySenses.cs

```diff
@@ -25,7 +25,7 @@ public class EnemySenses : MonoBehaviour
     {
         public static readonly Vector3 ResetPlayerPos = new Vector3(float.MaxValue, float.MaxValue, float.MaxValue);
 
-        public float SightRadius = 25f;         // Range of enemy sight
+        public float SightRadius = 4096 * MeshReader.GlobalScale;         // Range of enemy sight
```

Comment:
Wow! That's the radius of a whole city RMB block. Had no idea it was that large. Good job finding that. :)